ratatool-common/src/test/resources/big_sampler_avro_test_schema.avsc

```diff
@@ -0,0 +1,20 @@
+{
+    "type": "record",
+    "name": "BigSamplerAvroTestRecord",
```

Comment:
Can just rename to `FlatAvroSchema` or something that better describes the purpose. Any reason to not just use the `RequiredNestedRecord` in the other one?

Reply:
Good idea I'll use the required ones!